indexer/services/comlink/src/config.ts

```diff
@@ -60,6 +60,9 @@ export const configSchema = {
   // vaults table is added.
   EXPERIMENT_VAULTS: parseString({ default: '' }),
   EXPERIMENT_VAULT_MARKETS: parseString({ default: '' }),
+
+  // Affiliates config
+  VOLUME_ELIGIBILITY_THRESHOLD: parseInteger({ default: 10_000 }),
```

Comment:
Can you verify volume is stored in dollars and not quote quantums?

Reply:
I believe the quantum conversion is made before writing to fills db
```
    fill_amount = dydx_trim_scale(dydx_from_jsonlib_long(event_data->'fillAmount') *
                                  power(10, perpetual_market_record."atomicResolution")::numeric);
    maker_price = dydx_trim_scale(dydx_from_jsonlib_long(maker_order->'subticks') *
                                  power(10, perpetual_market_record."quantumConversionExponent" +
                                                     asset_record."atomicResolution" -
                                                     perpetual_market_record."atomicResolution")::numeric);
```
This screenshot also suggests it is converted 

<img width="552" alt="image" src="https://github.com/user-attachments/assets/10ecdf17-19b4-46f7-a009-8b22290d8cb2">